README.md

```diff
@@ -31,35 +31,48 @@ This method allows for using this image as an ambassador to other (secure) serve
 	docker stop staging-mongo;
 	docker rm staging-mongo;
 	docker run -d --name staging-mongo -v $SSH_AUTH_SOCK:/ssh-agent kingsquare/tunnel *:2222:127.0.0.1:27017 tunnel-user@db.staging
-	
+
 	docker stop production-mongo;
 	docker rm production-mongo;
 	docker run -d --name production-mongo -v $SSH_AUTH_SOCK:/ssh-agent kingsquare/tunnel *:2222:127.0.0.1:27017 tunnel-user@db.production
-	
+
 use the links in another container via exposed port 2222:
 
-	docker run --link staging-mongo:db.staging \ 
+	docker run --link staging-mongo:db.staging \
 	    --link production-mongo:db.production \
 	    my_app start
 
 # Changelog
 
+* 2017-01-10
+
+   - Modified image to use the `alpine:latest`
+   - Use `autossh` instead of simple `ssh` for extra stability of the tunnel
+   - Provided sample `Makefile` to automate the build process
+
+	```SSH_CMD="*:6379:localhost:6379 martin@172.17.0.1" make build-container```
+
+   - As original, the assumption is, that local `ssh-agent` holds the
+     required identity files.  Better solution probably would be to
+     generate new ssh key (`ssh-keygen`) and use the ssh `-i` option to
+     provide the identity directly
+
```

Comment:
In my deployments I am just used to import project specific keys -- I like to be explicit what keys are used (and make them part of the container) than to rely on local agent to provide the identities.   But I have no intention to change that in your repository, as I have not yet found good enough solution to generate and provide those keys -- it always requires a bit of shell scripting